test-suites/pool-flashloan.spec.ts

```diff
@@ -542,8 +581,48 @@ makeSuite('Pool: FlashLoan', (testEnv: TestEnv) => {
     expect(callerDebt.toString()).to.be.equal('500000000', 'Invalid user debt');
   });
 
+  it('Disable USDC borrowing. Caller deposits 5 WETH as collateral, Takes a USDC flashloan with mode = 2, does not return the funds. Revert creating borrow position (revert expected)', async () => {
+    const {usdc, pool, weth, configurator, users, helpersContract} = testEnv;
+
+    const caller = users[2];
+
+    expect(await configurator.setReserveStableRateBorrowing(usdc.address, false));
+    expect(await configurator.setReserveBorrowing(usdc.address, false));
+
+    let usdcConfiguration = await helpersContract.getReserveConfigurationData(usdc.address);
+    expect(usdcConfiguration.borrowingEnabled).to.be.equal(false);
+
+    await weth
+      .connect(caller.signer)
+      ['mint(uint256)'](await convertToCurrencyDecimals(weth.address, '5'));
+
+    await weth.connect(caller.signer).approve(pool.address, MAX_UINT_AMOUNT);
+
+    const amountToDeposit = await convertToCurrencyDecimals(weth.address, '5');
+
+    await pool.connect(caller.signer).deposit(weth.address, amountToDeposit, caller.address, '0');
+
+    await _mockFlashLoanReceiver.setFailExecutionTransfer(true);
```

Comment:
This is not needed, since it does not reach the `executeOperation` logic

Reply:
removed